crates/consensus/consensus/Cargo.toml

```diff
@@ -11,7 +11,8 @@ repository.workspace = true
 workspace = true
 
 [dependencies]
-reth-primitives.workspace = true
+# reth-primitives.workspace = true
+reth-primitives = { path = "../../primitives", default-features = false }
```

Comment:
since this will only work locally, better bring default features into scope. then you could open an issue for adding `default-features = false` in workspace, and adding features for deps in all crates explicitly. I was actually thinking of doing the same for many more deps.